mantidproject/mantid
# Make SNSPowderReduction work with various splitter workspaces

Make SNSPowderReduction work with various types of workspaces that contain splitters.

**To test:**

a1. create 1 splitters workspaces by GenerateEventsFilter.  Set 'FastLog' to True.  The splitter workspace should be a MatrixWorkspace
a2. run SNSPowderReduction with this MatrixWorkspace splitters.

b1. create a 3 column TableWorkspace. column 0 and 1 should be float, and column should be string. Column 0 should be the start time of a splitter; column 1 be the stop time; and column 3 be the target workspace tag.
b2. run SNSPowderReduction with this TableWorkspace.




Fixes #19139.

*New feature is reported in the release note.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
